README.md

```diff
@@ -24,7 +25,7 @@ RUSTC_WRAPPER=mirai cargo build
 The touch command (which needs to reference a real file in your project) forces Cargo to re-run rustc and to not assume
 that it's cached error messages are still correct.
 
-This will likely produce a lot of warnings, which you can then fix by adding annotations using this
+This will likely produce a lot of warnings, which you can then fix by adding annotations declared in this
  [crate](https://crates.io/crates/mirai-annotations). Keep running cargo build as above until there are no more warnings.
```

Comment:
I would change this to "Keep re-touching and running cargo build as above until there are no more warnings"